core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -448,7 +448,8 @@ enum class UniqueType(
     UnitUpgradeCost("[relativeAmount]% Gold cost of upgrading", UniqueTarget.Unit, UniqueTarget.Global),
 
     // Gains from battle
-    DamageUnitsPlunder("Earn [amount]% of the damage done to [combatantFilter] units as [civWideStat]", UniqueTarget.Unit, UniqueTarget.Global),
+    DamageUnitsPlunder("Earn [amount]% of the damage done to [combatantFilter] units as [stockpile/civWideStat]", UniqueTarget.Unit, UniqueTarget.Global),
+    DamageUnitsPlunderStockpile("Gain [amount]% of the damage done to [combatantFilter] units as [stockpile]", UniqueTarget.Unit, UniqueTarget.Global),
```

Comment:
These read, in English, as exactly the same thing. Are they different?

Reply:
Not necessarily, but for code clarity reasons, I decided to initially keep them separate. As written in my commit note, merging them is complicated, was waiting for the initial response before merging the two uniques